tests/Boascan.test.ts

```diff
@@ -70,18 +80,27 @@ describe("Test of Stoa API Server", () => {
         coinMarketService = new CoinMarketService(gecko_market);
     });
 
+    before("Start a fake votera Server and Service", () => {
+        return new Promise<void>(async (resolve, reject) => {
+            votera_server = new TestVoteraServer(1337, votera_addr, resolve);
+            votera_service = new VoteraService(votera_addr);
+        });
+    });
     before("Create TestStoa", () => {
         testDBConfig = MockDBConfig();
-        stoa_server = new TestStoa(testDBConfig, agora_addr, parseInt(stoa_addr.port, 10), coinMarketService);
+        stoa_server = new TestStoa(testDBConfig, agora_addr, parseInt(stoa_addr.port, 10), votera_service, coinMarketService);
         return stoa_server.createStorage();
     });
 
-    before("Start TestStoa", () => {
-        return stoa_server.start();
+    before("Start TestStoa", async () => {
+        await stoa_server.start();
+        await stoa_server.voteraService?.stop();
```

Comment:
Is there a reason to stop VoteraService?

Reply:
yes I dont want to start voteraService while my other testcases are running.